cmake/configs/default.cmake

```diff
@@ -272,7 +272,7 @@ hunter_config(odb-sqlite VERSION 2.4.0)
 hunter_config(ogles_gpgpu VERSION 0.2.8)
 hunter_config(onmt VERSION 0.4.1-p2)
 hunter_config(openddlparser VERSION 0.1.0-p2)
-hunter_config(opentracing-cpp VERSION 1.1.0)
+hunter_config(opentracing-cpp VERSION 1.3.0)
```

Comment:
Doesn't belong here, I guess you should rebase your change.

Reply:
woops! good catch. not sure how this one got in there